show/main.py

```diff
@@ -1848,9 +1848,9 @@ def ssdhealth(device, verbose, vendor):
 @click.option('-c', '--check', is_flag=True, help="Check the platfome pcie device")
 def pcieinfo(check, verbose):
     """Show Device PCIe Info"""
-    cmd = "pcieutil pcie_show"
+    cmd = "pcieutil pcie-show"
```

Comment:
Why?

Reply:
show platform pcieinfo
Usage: pcieutil [OPTIONS] COMMAND [ARGS]...
Try "pcieutil --help" for help.

Error: No such command "pcie_show".
admin@sonic:~$
 

pcieutil support pcie-show

Reply:
admin@sonic:~$ pcieutil --help
Usage: pcieutil [OPTIONS] COMMAND [ARGS]...

  pcieutil - Command line utility for checking pci device

Options:
  --help  Show this message and exit.

Commands:
  pcie-check     Check PCIe Device
  pcie-generate  Generate config file with current pci device
  pcie-show      Display PCIe Device
  version        Display version info
admin@sonic:~$